react/components/PickUpHeader.tsx

```diff
@@ -26,6 +26,8 @@ const StorePickUpHeader: FC<Props> = ({ shippingData, index, numPackages }) => {
   const multiplePickups = numPackages > 1
   const { receiverName } = shippingData.address
   const { additionalInfo } = shippingData.selectedSlaObj.pickupStoreInfo
```

Comment:
in theory you can remove this line and use it as `pickupStoreInfo.additionalInfo`, right? so that you don't have 2 imports of the same code

Reply:
Done :)